src/barretenberg/mod.rs

```diff
@@ -36,6 +35,8 @@ pub(crate) enum FeatureError {
     InvalidUsize { value: i32, source: std::num::TryFromIntError },
     #[error("Value expected to be 0 or 1 representing a boolean")]
     InvalidBool,
+    #[error("Black Box vendor Instance creation failed")]
+    BlackBoxInstanceFailed,
```

Comment:
Does this given us any benefit? We immediately unwrap this error anyway.

Reply:
Yes, we can get rid of this, I was going somewhere else with it...